test/basics.jl

```diff
@@ -1,19 +1,24 @@
 using Functors, Test
 
-@testset "Nested" begin
```

Comment:
Why remove the test set?

Reply:
it's not removed. some structs are now defined outside of a specifc testset and used in multiple testsets.